src/main/java/de/retest/web/RecheckSeleniumAdapter.java

```diff
@@ -22,13 +23,26 @@
 
 import de.retest.recheck.RecheckAdapter;
 import de.retest.ui.DefaultValueFinder;
+import de.retest.ui.descriptors.IdentifyingAttributes;
+import de.retest.ui.descriptors.RetestIdProvider;
 import de.retest.ui.descriptors.RootElement;
 import ru.yandex.qatools.ashot.AShot;
 import ru.yandex.qatools.ashot.shooting.ShootingStrategy;
 import ru.yandex.qatools.ashot.shooting.ViewportPastingDecorator;
 
 public class RecheckSeleniumAdapter implements RecheckAdapter {
 
+	// TODO Remove again after retest 3.0.1 release...
+	public final static RetestIdProvider idProvider = new RetestIdProvider() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Reorder the modifiers to comply with the Java Language Specification. ([squid:ModifiersOrderCheck](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AModifiersOrderCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=78&issues=AWaLhDVhcPetVqywBZ07&open=AWaLhDVhcPetVqywBZ07)

[ISSUE_KEY]: AWaLhDVhcPetVqywBZ07